waves-integration-it/src/test/scala/com/wavesplatform/dex/grpc/integration/clients/BlockchainUpdatesClientTestSuite.scala

```diff
@@ -147,7 +147,7 @@ class BlockchainUpdatesClientTestSuite extends IntegrationSuiteBase with HasToxi
       Thread.sleep(5.seconds.toMillis)
 
       cancellable.cancel()
-      val xs = Await.result(eventsF, 1.minute).map { evt =>
+      val xs = eventsF.futureValue.map { evt =>
```

Comment:
can we preserve original timeout?

Reply:
We could do this, but there is overrided PatienceConfig with the same timeout, so it's already preserved